### DISCUSSION TITLE
Yi 1080p Home IFUS is suddenly inaccessible

### DISCUSSION THREAD

**Discussion by cbroissa-github** [roleoroleo/yi-hack-Allwinner-v2#285] [Q&A]
Hello,
I have two cam Yi 1080p Home IFUS, with HA integration, it worked perfectly.
But since a few days, I can't access the cam in HA, and  http://ip-cam:8080/ is not reachable.
I can ping 'ip-cam'

My two others cam Yi Outdoor 1080p IFUS work in HA.

What could I do or check?
Thank you.

**Comment by roleoroleo** [comment]
Try to enter with ssh (putty) and check the list of running processes with `ps`.

**Reply by cbroissa-github** [reply]
here si the processes 

PID USER       VSZ STAT COMMAND
    1 root       980 S    /sbin/init
    2 root         0 SW   [kthreadd]
    3 root         0 SW   [ksoftirqd/0]
    5 root         0 SW<  [kworker/0:0H]
    7 root         0 SW   [rcu_preempt]
    8 root         0 SW   [rcu_sched]
    9 root         0 SW   [rcu_bh]
   10 root         0 SW<  [lru-add-drain]
   11 root         0 SW   [kdevtmpfs]
  210 root         0 SW   [oom_reaper]
  211 root         0 SW<  [writeback]
  213 root         0 SW   [kcompactd0]
  214 root         0 SW<  [crypto]
  215 root         0 SW<  [bioset]
  217 root         0 SW<  [kblockd]
  258 root         0 SW   [sys_user]
  266 root         0 SW<  [cfg80211]
  272 root         0 SW<  [watchdogd]
  287 root         0 SW<  [spi0]
  297 root         0 SW   [kswapd0]
  352 root         0 SW   [kworker/u2:1]
  380 root         0 SW<  [bioset]
  385 root         0 SW<  [bioset]
  390 root         0 SW<  [bioset]
  395 root         0 SW<  [bioset]
  400 root         0 SW<  [bioset]
  405 root         0 SW<  [bioset]
  410 root         0 SW<  [bioset]
  415 root         0 SW<  [bioset]
  420 root         0 SW<  [bioset]
  448 root         0 SW   [irq/302-sunxi-m]
  450 root         0 SW   [irq/166-sdc0 cd]
  451 root         0 SW   [irq/303-sunxi-m]
  511 root         0 SW<  [bioset]
  512 root         0 SW   [mmcqd/0]
  531 root         0 SW<  [kworker/0:1H]
  544 root         0 SWN  [jffs2_gcd_mtd4]
  551 root       980 S    -/bin/sh
  583 root       980 S    /usr/sbin/telnetd
  595 root         0 SW<  [phy0-atbm_wq]
  596 root         0 SW   [phy0-usb_atbm_b]
  639 root      1104 S    ./dispatch
 1176 root     51052 S    ./rmm
 1256 root      1400 S    /backup/tools/wpa_supplicant -c/tmp/wpa_supplicant.c
 1307 root       980 S    udhcpc -i wlan0 -b -s /backup/tools/default.script -
 1534 root       844 S    pure-ftpd (SERVER)
 1546 root       876 S    dropbear -R -B -p 0.0.0.0:22
 1548 root       668 S    ipc_multiplexer
 1595 root      5024 S<   rRTSPServer
 1608 root       988 S    {wd_rtsp.sh} /bin/sh /tmp/sd/yi-hack/script/wd_rtsp.
 1646 root      7280 S    onvif_srvd --conf_file /tmp/onvif_srvd.conf
 1668 root      2072 S    wsdd --pid_file /var/run/wsdd.pid --if_name wlan0 --
 1669 root      2420 S    framefinder y211ga
 2252 root         0 SW   [kworker/u2:0]
 3916 root         0 SW   [kworker/u2:2]
 4382 root       900 R    dropbear -R -B -p 0.0.0.0:22
 4418 root       980 S    -ash
 4463 root       980 S    sleep 10
 4468 root       980 R    ps
24588 root         0 SW   [kworker/0:2]
32163 root         0 SW   [kworker/0:1]

**Comment by roleoroleo** [comment]
The httpd process is missing.
Check the free space
`df`
and dump the configuration
`cat /tmp/sd/yi-hack/etc/system.conf`

**Comment by cbroissa-github** [comment]
Sorry for the delay, I was in holidays :)

Here is the check for free space :
Filesystem           1024-blocks    Used Available Use% Mounted on
/dev/root                 1280      1280         0 100% /
devtmpfs                 30400      1824     28576   6% /dev
/dev/mtdblock3            2816      2816         0 100% /home
/dev/mtdblock4            1216       960       256  79% /backup
tmpfs                    32768       128     32640   0% /tmp
/dev/mtdblock4            1216       960       256  79% /home/app/script/wifidhc                                                                                                                     p.sh
/dev/mmcblk0p1           32768       128     32640   0% /etc/passwd
/dev/mmcblk0p1           32768       128     32640   0% /etc/shadow
/dev/mmcblk0p1        30909472    136608  30772864   0% /tmp/sd
tmpfs                    32768       128     32640   0% /etc
/dev/mmcblk0p1        30909472    136608  30772864   0% /tmp/sd/yi-hack/www/record

And the dump

HTTPD=yes
TELNETD=yes
SSHD=yes
FTPD=yes
BUSYBOX_FTPD=no
DISABLE_CLOUD=yes
REC_WITHOUT_CLOUD=no
MQTT=no
RTSP=yes
RTSP_STREAM=high
RTSP_AUDIO=
SPEAKER_AUDIO=yes
ONVIF=yes
ONVIF_WSDD=yes
ONVIF_PROFILE=high
ONVIF_NETIF=wlan0
ONVIF_WM_SNAPSHOT=yes
NTPD=yes
NTP_SERVER=pool.ntp.org
PROXYCHAINSNG=no
SWAP_FILE=no
RTSP_PORT=554
ONVIF_PORT=80
HTTPD_PORT=8080
USERNAME=USERNAME
PASSWORD=PASSWORD
TIMEZONE=CET-1CEST,M3.5.0,M10.5.0/3
FREE_SPACE=0
FTP_UPLOAD=no
FTP_HOST=
FTP_DIR=
FTP_DIR_TREE=no
FTP_USERNAME=
FTP_PASSWORD=
FTP_FILE_DELETE_AFTER_UPLOAD=yes
SSH_PASSWORD=SSH_PASSWORD
CRONTAB=

Thank you for your time with my problem.

**Comment by roleoroleo** [comment]
Free space is ok.
About config, fix RTSP_AUDIO field adding a value:
`RTSP_AUDIO=alaw`

Finally, try to start httpd manually:
`httpd -p 8080 -h /tmp/sd/yi-hack/www/ -c /tmp/httpd.conf`

**Reply by cbroissa-github** [reply]
when I start httpd 

-ash: httpd: not found

**Reply by roleoroleo** [reply]
You have to source the env:
`. /tmp/sd/yi-hack/script/env.sh`

**Reply by cbroissa-github** [reply]
ok done, but now I have this with the httpd command
Segmentation fault (core dumped)

The same message on my two cameras.

**Reply by roleoroleo** [reply]
Probably the binary is corrupted.
Maybe the sd card is not ok?
Try to create again the sd from scratch.

**Reply by cbroissa-github** [reply]
ok I have empty the card and put the release and it's now ok
